frontend/src/components/common/InviteButton.vue

```diff
@@ -109,6 +103,11 @@ const isOptionUnselectable = (optionName: string) => {
   return optionName === 'Read';
 };
 
+//Validation
+const schema = object({
+  Email: string().matches(new RegExp(Regex.EMAIL), 'Provide a valid email address')
+});
```

Comment:
we should do: 'if restrict toggle is on, email field is required' rule
we have an example of how to implement this in previous code.

Reply:
it will be something like this:
```js
const schema = object({
      isRestricted: boolean(),
      Email: string().when("isRestricted", {
        is: true,
        then: string().matches(new RegExp(Regex.EMAIL), 'Provide a valid email address'),
      }),
    });
 ```